terra-jupyter-hail/scripts/kernel/python_kernelspec.tmpl

```diff
@@ -0,0 +1,15 @@
+{
+ "language": "python",
+ "display_name": "Python ${PY_VERSION}",
```

Comment:
just curious..where is `PY_VERSION` set?

Reply:
`kernelspec.sh` replaces it with "3": https://github.com/DataBiosphere/terra-docker/pull/99/files#diff-d7fa6e149ccb1fc5c30017df315a618cR7

I sort of want to remove `kernelspec.sh` -- now that we have custom dockers (and fewer kernels) we could just `ADD` the `kernel.json`s in the right places directly in the Dockerfile. But changing that pattern would touch the base image + init-actions.sh so I felt it's a bit out of scope of this PR.